cli/src/plugin_window-status.js

```diff
@@ -0,0 +1,16 @@
+var waitForWindowStatus = function(desiredStatus) {
+    return new Promise(function(resolve) {
+        var poller = setInterval(function() {
+            if (window.status && window.status == desiredStatus) {
+                clearInterval(poller);
+                resolve(desiredStatus);
+            }
+        }, 1000);
```

Comment:
1000 seems like too much. Probably should go with something like 100-200?

Reply:
hmm, I think you are right. 100 should be more than enough.